code/modules/power/supermatter/supermatter.dm

```diff
@@ -138,8 +140,10 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	var/tritiumcomp = 0
 	var/bzcomp = 0
 	var/pluoxiumcomp = 0
+	var/freoncomp = 0
 
 	var/pluoxiumbonus = 0
+	var/freonbonus = 1 //it allows us to remove the power generation while freon is inside the chamber
```

Comment:
The goal here is to stunt all of the gasses effects right? I don't quite like the clutter but it's not a big deal

Reply:
i've changed where the bonus will affect. power_transmission_bonus was left as it was before. I moved the freonbonus to the radiation calculation